src/plugins/discover/public/application/main/components/layout/discover_documents.tsx

```diff
@@ -121,7 +121,7 @@ function DiscoverDocumentsComponent({
   }
 
   return (
-    <EuiFlexItem className="dscTable" tabIndex={-1} aria-labelledby="documentsAriaLabel">
```

Comment:
So, removing the the tabIndex fixes the page navigation issue, now we need to find out: Why was it used here? Since we don't wanna break something else?

Reply:
tabIndex war introduced in #103119

Reply:
tabIndex made sense in #103119 but due to many changes in this area it should be redundant

Reply:
Ok it seems we broke "Move focus on chart toggle in Discover" - #103119 a while ago, so this PR doesn't double break this, and we should fix it in a future PR. But as you mentioned now `onBackToTop` doesn't work like it worked before, good call, we might think of moving focus to another element.

Reply:
We could focus the "Go to end of the table" button to focus when using the "Back to top" button, think this would also be handy if the user doesn't like the start of the the table :)
<img width="190" alt="Bildschirmfoto 2022-01-12 um 18 21 39" src="https://user-images.githubusercontent.com/463851/149190296-192be7dd-6949-43d8-b3c3-62e87a54693c.png">

Reply:
did that and also improved the functional testing to catch it if it's broken again